spec/responses/fastlane_releases.json

```diff
@@ -0,0 +1,1112 @@
+[
+  {
+    "url": "https://api.github.com/repos/fastlane/fastlane/releases/2112556",
+    "assets_url": "https://api.github.com/repos/fastlane/fastlane/releases/2112556/assets",
+    "upload_url": "https://uploads.github.com/repos/fastlane/fastlane/releases/2112556/assets{?name,label}",
+    "html_url": "https://github.com/fastlane/fastlane/releases/tag/1.39.0",
+    "id": 2112556,
+    "tag_name": "1.39.0",
+    "target_commitish": "master",
+    "name": "1.39.0 New Actions and Features",
+    "draft": false,
+    "author": {
+      "login": "KrauseFx",
+      "id": 869950,
+      "avatar_url": "https://avatars.githubusercontent.com/u/869950?v=3",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/KrauseFx",
+      "html_url": "https://github.com/KrauseFx",
+      "followers_url": "https://api.github.com/users/KrauseFx/followers",
+      "following_url": "https://api.github.com/users/KrauseFx/following{/other_user}",
+      "gists_url": "https://api.github.com/users/KrauseFx/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/KrauseFx/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/KrauseFx/subscriptions",
+      "organizations_url": "https://api.github.com/users/KrauseFx/orgs",
```

Comment:
Can we reduce the response to a minimum, e.g. only 1 or 2 releases, so that we don't have so many extra lines?